.github/workflows/ci-pre-commit.yml

```diff
@@ -59,7 +59,10 @@ jobs:
         working-directory: contracts/evm
         run: forge install nomad-xyz/ExcessivelySafeCall@be417ab0c26233578b8d8f3a37b87bd1fcb4e286 --no-git --no-commit
       - name: Install Anchor CLI
-        run: npm install -g @coral-xyz/anchor-cli@v0.30.1
+        run: |
+          cargo install --git https://github.com/coral-xyz/anchor avm
+          avm install latest
+          avm use latest
```

Comment:
the previous way doesn't currently work 
https://github.com/coral-xyz/anchor/issues/3596